From f4f107df96f7ebfd4916a45e211fc73af481a3e1 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 18 May 2023 16:32:49 +0530
Subject: [PATCH] WA-Fixed Build Error For Python.

Configuring python2 for now to resolve build issue.

Tracked-On: OAM-OAM-106853
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 path/linux-x86/python | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/path/linux-x86/python b/path/linux-x86/python
index 94246bc..8ec5dcd 120000
--- a/path/linux-x86/python
+++ b/path/linux-x86/python
@@ -1 +1 @@
-../../linux-x86/bin/py3-cmd
\ No newline at end of file
+../../linux-x86/bin/py2-cmd
\ No newline at end of file
-- 
2.17.1

